components/newMentorProfile/sessions.js

```diff
@@ -0,0 +1,107 @@
+import React,{ useState } from 'react'
+import SessionCard from "../../components/mentorProfile/components/SessionCard";
+import styles1 from "../../styles/mentorTestimonial.module.css";
+function Sessions({mentorDetail}) {
+  const [modalPopup, setModalPopup] = useState(false);
```

Comment:
I think this is not the component, the Session card is the component. 
May we can have this is `[mentorUsername].page.js`as before.

Reply:
The session card component is imported inside the sessions.js ,which is then placed in mentor username as a component.